styles/output/nursing-internal-pdf.css

```diff
@@ -1805,42 +1805,37 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
   margin-left: auto;
   margin-right: auto;
 }
-
 .os-table:not(.os-unstyled-container) > table {
   border-collapse: collapse;
   width: 100%;
   margin-bottom: 0;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th {
-  border-bottom: 0.1rem solid black;
-  border-top: 0.1rem solid black;
+  border-bottom: 0.1rem solid #000000;
+  border-top: 0.1rem solid #000000;
   font-family: Mulish, sans-serif;
   font-weight: 700;
   padding: 0.7rem;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-valign=top] {
   vertical-align: top;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-align=left] {
   text-align: left;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-align=right] {
   text-align: right;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-align=center] {
   text-align: center;
 }
 
 .os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td {
-  border-top-color: black;
+  border-top-color: #000000;
+  font-family: Mulish, sans-serif;
```

Comment:
ah, and looks like a font here was added where it wasn't previously

Reply:
added comment